README.md

```diff
@@ -29,25 +29,31 @@ and documentation, who to contact about what, etc.
 go get -u golang.org/x/vgo
 ```
 
-### Set up protobufs
+### Install non-go development dependencies
 
-In order to develop on storj, you will need to have protobufs and gRPC installed on your system.
+In order to develop on storj, you will need to have the protobuf compiler installed on your system.
 
 1. Grab the latest release for your system from [here](https://github.com/google/protobuf/releases)
 
-2. place the `protoc` binary in your path. i.e
+1. place the `protoc` binary in your path. i.e
```

Comment:
2 ?

Reply:
I intentionally set them all to "1."; github does the right thing, and this way we can insert/move things around without having to worry about the numbers. We get the editing benefit of an un-ordered list, in an ordered list :D.